library/liquibase

```diff
@@ -0,0 +1,7 @@
+Maintainers: Jake Newton <docker@liquibase.com> (@jnewton03)
+GitRepo: https://github.com/liquibase/docker.git
+
+Tags: 4.23, 4.23.1, 4.23-alpine, 4.23.1-alpine, latest, latest-alpine
```

Comment:
Can you remove the `latest-alpine`. We usually only have one latest for the latest version of the default OS